mac

```diff
@@ -86,7 +86,6 @@ gem_install_or_update() {
     gem update "$@"
   else
     gem install "$@"
-    rbenv rehash
```

Comment:
It hasn't proven necessary for me just yet, but I'm not sure if that's because of the way I set up this new laptop in steps debugging this script or what.

Do you know it to be required?